packages/cli/templates/default-backend-plugin/tsconfig.json

```diff
@@ -1,7 +1,7 @@
 {
   "extends": "@backstage/cli/config/tsconfig.json",
   "include": [
-    "src",
+    "src2",
```

Comment:
Whaa

Reply:
oh wow, that's VSCode updating that under the hood as I was renaming to keep the old template as a reference